sema/commons/rdf_ogm/graph_builder.py

```
@@ -0,0 +1,7 @@
+class GraphBuilder:
+    def __init__(self):
+        self. graph = ...
+        self._build()
+    
+    def _build(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_build`

Add return type annotation: `NoReturn`

(ANN202)